native/js/Scaffold.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import HappoJob from './HappoJob';
+
+class Scaffold extends Component {
```

Comment:
Stateless function components? Looks like it should work: https://medium.com/front-end-hacking/stateless-components-in-react-native-e9034f2e3701#.z3v1kwmvf